File: python/ejer36.py
#solicitar N numeros y contar cuantos son positivos 
def contar_positivos(numeros):
    contador = 0
    for numero in numeros:
        if numero > 0:
            contador += 1
    return contador 
def main():
    numeros = []
    n = int(input("Ingrese la cantidad de números que desea ingresar: "))
    for i in range(n):
        numero = float(input(f"Ingrese el número {i+1}: "))
        numeros.append(numero)
    cantidad_positivos = contar_positivos(numeros)
    print(f"La cantidad de números positivos es: {cantidad_positivos}") 
if __name__ == "__main__":    main()

File: js/ejer36.js
// solicitar N numeros y contar cuantos son positivos 
let cantidadNumeros = parseInt(prompt("Ingrese la cantidad de numeros que desea ingresar:"));
let contadorPositivos = 0;

for (let i = 0; i < cantidadNumeros; i++) {
    let numero = parseInt(prompt("Ingrese un numero:"));
    if (numero > 0) {
        contadorPositivos++;
    }
}
document.write("La cantidad de numeros positivos ingresados es: " + contadorPositivos); 

